nextcloudappstore/core/forms.py

```diff
@@ -50,27 +51,40 @@ class AppRegisterForm(Form):
                               'openssl base64</b>'))
     safe_help_fields = ['certificate', 'signature']
 
+def get_languages_local(language=None):
+    if language:
+        languages = list(language)
+    else:
+        languages = [ l[0] for l in settings.LANGUAGES]
```

Comment:
This seems to be used in a choice field. Isn't a choice field a tuple of code and language?